### PR TITLE
Ahmed

### LINKED ISSUE #1: FR1: Splash Screen
|  FR1. Splash Screen  |
|-----------------------|
| Goal: FR1 opens the app and the user will be greeted with our logo. |
| Stakeholders: All users  |
| Description: When a user starts up the app, they will see a splash screen which contains our logo. This splash screen will be displayed while the app components are loading.  |
| Origin: Team meeting on 1/31 |
| Version: 1.0 | 
| Date: 01/31/2023 |
| Priority: 3 |

### LINKED ISSUE #8: FR8. Navigation bar
|  FR8. Navigation bar  |
|-----------------------|
| Goal: FR8 is a simple bar for the user to switch between pages of the app |
| Stakeholders: All Users |
| Description: A static bar at the bottom of the screen with buttons to move to the homepage, the settings page, or the financial tips page. This bar will also highlight the page the user is currently viewing. |
| Origin: Team meeting on 1/31 |
| Version: 1.0 | 
| Date: 01/31/2023 |
| Priority: 1 |

### LINKED ISSUE #14: FR14. Settings Page
| FR14. Settings Page   |
|-----------------------|
| Goal: FR14 lists the settings for the application. |
| Stakeholders: All users |
| Description: This page will be located on the right-most side of the navigation bar. The user will be able to change parts of the application to their desire. |
| Origin: Team meeting on 1/31 |
| Version: 1.0 | 
| Date: 01/31/2023 |
| Priority: 3 |

### PR DESCRIPTION
**Pull request by Ahmed-Alagbari**
FR1 and FR14. For FR14 I added some account settings as placeholders until we work on the login process

### CODE REVIEW

**Review by Rey-Jey — APPROVED** [review]
Good work with the splash screen and the settings page. I think the splash screen looks good for now, but if you'll allow me, I would like to change the font of the name at a later date.

**Review by JayanthAnandNama — APPROVED** [review]
LGTM!

File: android/app/src/main/res/drawable-v21/background.png
Comment (amuwhao46) [inline android/app/src/main/res/drawable-v21/background.png:1]: Will this also appear on iOS devices?

File: android/app/src/main/res/drawable/launch_background.xml
Comment (amuwhao46) [inline android/app/src/main/res/drawable/launch_background.xml:1]: I don't know how important this is but have you tested this on other device sizes as well?

File: lib/settingPage.dart [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+}
+
+Widget buildLogout()=>SimpleSettingsTile(
+  title:'Logout',
+  subtitle: '',
+  leading:const Icon(Icons.logout)
+
+);
+
+Widget buildDeleteAccount()=>SimpleSettingsTile(
```
Comment (amuwhao46) [inline lib/settingPage.dart:52]: Looks like we'll need to develop functionality if we decide to implement this

**Review by amuwhao46 — APPROVED** [review]
After looking through all of the code everything looks good to me. Only thing I'm thinking about is the delete account function in the settings file. We can discuss the during the next scrum meeting

File: android/app/src/main/res/drawable-v21/background.png
Comment (Ahmed-Alagbari) [reply android/app/src/main/res/drawable-v21/background.png:1]: As far as I know, it should but I'm not able to test it. I don't have an Apple device

File: android/app/src/main/res/drawable/launch_background.xml
Comment (Ahmed-Alagbari) [reply android/app/src/main/res/drawable/launch_background.xml:1]: The way I implemented the splash screen makes it fill to the size of the screen of the device

File: lib/settingPage.dart (hunk 1 above)
Comment (Ahmed-Alagbari) [reply lib/settingPage.dart:52]: You are correct, I only added these two widgets as placeholders until the time comes

**Pull request merged**